ui/src/consts/difficultyLevel.ts

```diff
@@ -0,0 +1,7 @@
+import i18next from '../i18n';
+
+export const difficultyLevel = {
+  1: i18next.t('tasks.difficultyLevels.easy'),
```

Comment:
lepsze by było coś typu: 
```suggestion
  easy: {
    id: 1,
    label: i18next.t('tasks.difficultyLevels.easy'),
   }
```

Obiekt którego kluczami są cyfry to trochę jaka taka inaczej zapisana arrajka co jest dosyć dziwne.